subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -450,6 +504,7 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
 	SHELL_CMD(status, NULL, "Status of the Wi-Fi interface", cmd_wifi_status),
 	SHELL_CMD(statistics, NULL, "Wi-Fi interface statistics", cmd_wifi_stats),
 	SHELL_CMD(ap, &wifi_cmd_ap, "Access Point mode commands", NULL),
+	SHELL_CMD(reg_domain, NULL, "Set or Get Wi-Fi regulatory domain", cmd_wifi_reg_domain),
```

Comment:
On second thought, given that the command takes parameters, the help string could be a bit more specific (i. e. mention the command format and expected arguments).